### PR TITLE
Add device_path support

### PR DESCRIPTION
**Pull request by seremis**
I hope I'm doing this right, this is the first time I've submitted a pull request like this..

Some initial testing shows that the device_path parameter is working and opens the correct camera.  The device_path parameter takes precedence over the device_id parameter.

As discussed in #4

### CODE REVIEW

File: include/cv_camera/capture.h [hunk 1]
```diff
@@ -50,6 +50,21 @@ class Capture
...
+   * @throw cv_camera::DeviceError device open failed
+   */
+  void open(std::string device_path);
+
+  /**
+   * @brief Load camera info from file.
+   *
+   * This loads the camera info from the file specified in the camera_info_url parameter.
+   */
+  void load_camera_info();
```
Comment (OTL) [inline include/cv_camera/capture.h]: Could you keep ROS C++ style?
```
loadCameraInfo()
```

File: src/driver.cpp [hunk 2]
```diff
@@ -37,10 +38,14 @@ void Driver::setup()
...
                             PUBLISHER_BUFFER_SIZE,
                             frame_id));
-  if (private_node_.getParam("file", file_path) &&
-      file_path != "")
+
+  if (private_node_.getParam("file", file_path) && file_path != "")
   {
     camera_->openFile(file_path);
+  } else if (private_node_.getParam("device_path", device_path) &&
+      device_path != "")
```
Comment (OTL) [inline src/driver.cpp]: Fix indent?
```
} else if (private_node_.getParam("device_path", device_path) &&
           device_path != "")
```

File: include/cv_camera/capture.h [hunk 3]
```diff
@@ -50,6 +50,21 @@ class Capture
   void open(int32_t device_id);
 
   /**
+   * @brief Open capture device with device name.
+   *
+   * @param device_path path of the camera device
+   * @throw cv_camera::DeviceError device open failed
+   */
+  void open(std::string device_path);
```
Comment (OTL) [inline include/cv_camera/capture.h]: Can you use const `std::string &` instead of `std::string`?
```
void open(const std::string &device_path);
```

**Comment by seremis** [comment]
I implemented the requested changes in fa1269d

**Pull request merged**